src/components/pages/search.js

```diff
@@ -1,54 +1,67 @@
 import firstBy from 'thenby';
 import PropTypes from 'prop-types';
 import urls from '../../utils/urls.js';
+import SearchBar from '../search-bar.js';
+import Logger from '../../common/logger.js';
 import { useTranslation } from 'react-i18next';
 import { Table, Tag, Select, Form } from 'antd';
-import React, { useMemo, useState } from 'react';
 import { useRequest } from '../request-context.js';
 import { SearchOutlined } from '@ant-design/icons';
-import { useService } from '../container-context.js';
+import React, { useEffect, useState } from 'react';
+import { useDateFormat } from '../language-context.js';
+import { handleApiError } from '../../ui/error-helper.js';
 import { useGlobalAlerts } from '../../ui/global-alerts.js';
+import LanguageFlagAndName from '../language-flag-and-name.js';
+import { useSessionAwareApiClient } from '../../ui/api-helper.js';
 import { searchResultShape } from '../../ui/default-prop-types.js';
-import { useDateFormat, useLanguage } from '../language-context.js';
-import LanguageNameProvider from '../../data/language-name-provider.js';
-import CountryFlagAndName from '../localization/country-flag-and-name.js';
+import SearchApiClient from '../../api-clients/search-api-client.js';
+
+function sortDocs(docs) {
+  return docs
+    .map(doc => ({
+      ...doc,
+      tags: [...new Set(doc.tags)], // Some docs have duplicate tags we don't want to render
```

Comment:
I know this code was there before, but I think now we can remove the comment since we ran the script to reduce duplicates.

Reply:
Yes, removed, and that started a chain of more simplifications on the page.

Reply:
Oh nice, especially with gathering the filtering + sorting of docs in one place and simplifying it. Was a bit scattered and complicated before..